File: 08.03/desafioE.py
#!/usr/bin/python
'''
diferenca menor que 10^-6 - empate draw

'''
def main():
    a = input()
    b = input()
    c = input()
    d = input()

    if((a+b)%2 == (c+d)%2):
        print('black')
    else:
        print('white')


main()

File: 08.03/desafioE.c
#include<stdio.h>

int main(){
    int a, b, c, d;

    scanf("%d %d",&a, &b);
    scanf("%d %d",&c,&d);

    if((a+b)%2==(c+d)%2){
        printf("black\n");
    }else{
        printf("white\n");
    }
}